browser/css/jsdialogs.css

```diff
@@ -854,6 +854,7 @@ input[type='number']:hover::-webkit-outer-spin-button {
 }
 
 /* toolbox */
+.no-label.has-dropdown,
 .jsdialog.toolbox {
```

Comment:
This seems to be unrelated to this commit. Could you please add a different one with that change in and with the respective commit message?